### DISCUSSION TITLE
WinMerge use question...

### DISCUSSION THREAD

**Discussion by Jayshua83** [WinMerge/winmerge#944] [Q&A] [answered]
I have used WinMerge for a long time but I haven't figured a few things out. I was wondering if anyone had any tips or knew of options I needed to change? I have 2 files. A & B. WinMerge doesn't correctly compare them like Notepad++ or KDiff does.

What am I missing?

Thanks in Advance!

File A Line 9816 should compare to File B 9800 but it doesn't compare correctly leaving a whole mess...

[A.txt](https://github.com/WinMerge/winmerge/files/7073080/A.txt)
[B.txt](https://github.com/WinMerge/winmerge/files/7073079/B.txt)

**Comment by sdottaka** [comment, marked as answer]
I think it is due to the difference in Diff algorithm.

Try changing the Diff algorithm to histogram in the Compare/General category of the Options dialog.
 
![image](https://user-images.githubusercontent.com/98126/131268209-09d97484-cfcd-4693-9cce-f73f7854d249.png)

**Comment by Jayshua83** [comment]
Thank you! :) Never knew what all the options did !
